.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "tests/cbmc/aws-verification-model-for-libcrypto"]
 	path = tests/cbmc/aws-verification-model-for-libcrypto
 	url = https://github.com/awslabs/aws-verification-model-for-libcrypto.git
+[submodule "tests/litani"]
```

Comment:
Is this where we want to put it? Litani isn't exactly a test. Maybe `/tools/litani`?